t/templates/main.mustache

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<html lang="{{ lang }}">
+    <head>
+        <title>{{ title }}</title>
+        <meta charset="UTF-8" />
+        <meta description="This is a new template"/>
+        <style>
+        /* code gets the browser-default font
+         * kbd gets a slightly less common monospace font
+         * samp gets the hard pixelly fonts
+         */
```

Comment:
I don't think we need these comments up in there. Some sites can be high-traffic ones (like docs.perl6.org) and there's no reason for them to pay for extra bandwidth due to some fairly useless comments in markup.

Reply:
But this is a test file. I don't think it matters too much.

Reply:
That comment was meant to go to the main template and half the comments were already removed https://github.com/perl6/Pod-To-HTML/pull/47/commits/5259257f898f07331dae48021187e7a1ec787530